packages/components/src/internal/components/entities/SampleAliquotViewSelector.tsx

```diff
@@ -26,7 +26,7 @@ interface Props {
     allLabel?: string;
     headerLabel?: string;
     samplesLabel?: string;
-    updateAliquotFilter: (newMode?: ALIQUOT_FILTER_MODE) => void;
+    updateAliquotFilter: (newMode: ALIQUOT_FILTER_MODE) => void;
```

Comment:
nit: The associated `SampleAliquotViewSelector.spec.tsx` should be converted to RTL with these changes.